website/docs/reference/commands/test.md

```diff
@@ -28,4 +29,48 @@ dbt test --select "one_specific_model,test_type:singular"
 dbt test --select "one_specific_model,test_type:generic"
 ```
 
-For more information on writing tests, see the [Testing Documentation](/docs/build/data-tests).
+For more information on writing tests, see the [Testing Documentation](/docs/build/tests).
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.8">
+
+`dbt test` runs data tests and unit tests defined on models, sources, snapshots, and seeds. It expects that you have already created those resources through the appropriate commands.
+
+The tests to run can be selected using the `--select` flag discussed [here](/reference/node-selection/syntax).
+
+```bash
+# run data and unit tests
+dbt test
+
+# run only data tests
+dbt test --select test_type:data
```

Comment:
Oooh can we update this page too with the new `test_type`s -> https://docs.getdbt.com/reference/node-selection/methods#the-test_type-method

Reply:
I'll add it as a fast follow to this https://github.com/dbt-labs/docs.getdbt.com/issues/4889